### ISSUE TITLE
Remove -C param from command line

### ISSUE THREAD

**Issue by julie-sullivan** [opencb/cellbase#476]
We should remove the `-C` configuration flag from the command line. This flag allows users to specify a custom configuration file location. 

However, in `GenericRestWSServer`, the configuration file is hard coded to be in `/config`. 

Is this flag ever used anyway?

**Comment by julie-sullivan** [comment]
While we are at it, can we remove the `verbose` option too?

```
[WARNING] /home/julie/git/cellbase/cellbase-app/src/main/java/org/opencb/cellbase/app/cli/admin/executors/IndexCommandExecutor.java:[32,92] verbose in org.opencb.cellbase.app.cli.CliOptionsParser.CommonCommandOptions has been deprecated
```

**Comment by imedina** [comment]
That's correct, `-C` or `--conf` has never really used and new REST server gets CELLBASE_HOME parameter and assumes `/conf`

The same applies to `-v`, it has been never used.

Let's remove them, @javild are you ok with this?

**Comment by javild** [comment]
No. I would keep it. We have actually used it with the variant-annotation command, for example. It's convenient if you don't want to rely on the built-in configuration. The GenericRestWSServer is not meant to use this as it is run by the server.

**Comment by julie-sullivan** [comment]
@imedina @javild Then I could just remove the `-C` option from the `server` only? I don't like having it there if it's ignored.
